clusterbuster

```diff
@@ -863,10 +863,17 @@ function generate_environment() {
 
 function namespace_policy() {
     local policy
-    policy=$(call_api -w "$requested_workload" -s namespace_policy)
+    if [[ $(type -t "deployment_type_${deployment_type,,}_policy") = function ]] ; then
+        policy=$("deployment_type_${deployment_type,,}_policy")
+    else
+        policy=$(call_api -w "$requested_workload" -s namespace_policy)
+    fi
```

Comment:
I'm not entirely keen on this, and would like to do it within the API framework, but that's going to take some more thought so this is OK for now.  The deployment type policy function could get the namespace policy from the desired workload if desired.